File: ClientApp/src/app/models/AtributesGroup.ts
import {GroupType} from './GroupTypes';
import {Atribute} from './Atribute';
//type AtributesGroup={
//  idAtrbutesGroup: number
//}
export class AtributesGroup {  
  public atributes: Array<Atribute>;
  public idAtrbutesGroup?: number;
  public name?: string;
  //constructor(atributes: Array<Atribute>,  idAtrbutesGroup?: number, name?: string)
  //constructor(atributesGroup: AtributesGroup)
  constructor(params:AtributesGroup = {}as AtributesGroup){
    let{
      atributes = [],
      idAtrbutesGroup = undefined,
      name = undefined
    } = params;
    this.atributes = atributes;
    this.idAtrbutesGroup = idAtrbutesGroup;
    this.name = name;
  }
}

File: Models/Data/Atributes/AtributesGroup.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace ProductsReviewsAngular.Models
{
    public class AtributesGroup
    {
        [Display(Name = "№ Категории")]
        public int idAtrbutesGroup { get; set; }

        [Required(ErrorMessage = "Введите название категории")]
        [Display(Name = "Название категории")]
        [ConcurrencyCheck]
        [MaxLength(50)]
        public string name { get; set; }
        public virtual int idGroupType { get; set; }
        public virtual GroupType GroupType { get; set; }

        [Display(Name = "Созданные атрибуты")]
        public virtual List<Atribute> atributes { get; set; } = new List<Atribute>();

        public override string ToString()
        {
            if (GroupType != null)
            {
                return "id: " + idAtrbutesGroup + " name: " + name + " AtributeCount: " + atributes.Count() + " GroupTypeID:" + idGroupType + " GroupType:" + GroupType.ToString();
            }
            else
            {
                return "id: " + idAtrbutesGroup + " name: " + name + " AtributeCount: " + atributes.Count() + " GroupTypeID:" +idGroupType;
            }
        }
    }
}
